Content.Shared/Traits/Assorted/Components/FootstepVolumeModifierComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Traits.Assorted.Components;
+
+/// <summary>
+///     This is used for any trait that modifies footstep volumes.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class FootstepVolumeModifierComponent : Component
+{
+    /// <summary>
+    ///     What to add to the volume of sprinting, in terms of decibels.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float SprintModifier { get; set; } = 0f;
```

Comment:
```suggestion
    public float SprintVolumeModifier;
```

Reply:
I forgot to make the setter private, as I temporarily made it public for testing. [d8464f5](https://github.com/Simple-Station/Einstein-Engines/pull/658/commits/d8464f5dcbee110c28c8d38fdc16143e1a728ae4)

Reply:
Never mind, that causes the build to fail. [29fc22d](https://github.com/Simple-Station/Einstein-Engines/pull/658/commits/29fc22d7efd08a357aae1a0fe0699c15455ff380)